src/app_task.cpp

```diff
@@ -356,3 +386,67 @@ void AppTask::DispatchEvent(const AppEvent &event)
 		LOG_INF("Event received with no handler. Dropping event.");
 	}
 }
+
+void AppTask::UpdateTemperatureClusterState(const AppEvent &)
+{
+	static constexpr int16_t kTemperatureMeasurementAttributeMaxValue = 0x7fff;
```

Comment:
I have taken these values from weather station app. Not sure it these are really needed, especially for the webinar demo.